src/status_im/chat/styles/message/message.cljs

```diff
@@ -123,7 +136,7 @@
    :background-color styles/color-white})
 
 (def command-request-from-text
-  (merge style-sub-text {:margin-bottom 2}))
+  (assoc style-sub-text :margin-bottom 2))
```

Comment:
I'd suggest not to make these kinds of changes unless they are part of the objective of the PR. By bundling them with this PR it becomes harder for reviewers to find/understand actual changes.

Reply:
yes, better to not touch code without need

Reply:
This change was requested earlier and I wanted to keep the file consistent.